correlation_context/HTTP_HEADER_FORMAT.md

```diff
@@ -10,24 +10,64 @@ A correlation context header is used to pass the name-value context properties f
 
 `Correlation-Context`
 
-Multiple correlation context headers are allowed. Values can be combined in a single header according to the [RFC 7230](https://tools.ietf.org/html/rfc7230#page-24).
-
 ## Header value
 
-`name1=value1[;properties1],name2=value2[;properties2]`
+The header value contains a list of key/value pairs. The first pair MUST contain the version of the header
```

Comment:
should we allow to assume the default version to be `v=0`? I see why having versioning strategy defined early is a good thing. But assuming that format will be around for a while and we do not have specific plans for the next version - we can save on serialization/deserialization and potential user errors now.